### PR TITLE
Supressing warning messages on Netty deprecated Handler object

### PR DESCRIPTION
**Pull request by mrsaldana**
- [x] I have considered the risk of behavior change or other zero migration impact (https://github.com/OpenLiberty/open-liberty/wiki/Behavior-Changes).
- [x] If this PR fixes an Issue, the description includes "Fixes #FILLMEIN" or "Resolves #FILLMEIN" (verify `release bug` label if applicable: https://github.com/OpenLiberty/open-liberty/wiki/Open-Liberty-Conventions).
- [x] If this PR resolves an external Known Issue (including APARS), the description includes "Fixes #FILLMEIN" or "Resolves #FILLMEIN".

### CODE REVIEW

**Review by isaacrivriv — DISMISSED** [review]
Looks good, thanks for working on this!

**Review by isaacrivriv — DISMISSED** [review]
LGTM

File: dev/com.ibm.ws.transport.http/src/com/ibm/ws/http/netty/pipeline/HttpPipelineInitializer.java [hunk 1]
```diff
@@ -53,6 +53,8 @@
 import io.netty.handler.ssl.SslHandler;
```
Comment (pnicolucci) [inline dev/com.ibm.ws.transport.http/src/com/ibm/ws/http/netty/pipeline/HttpPipelineInitializer.java:53]: Adding a date range to the copyright header to include 2024 makes sense to me here.

File: dev/com.ibm.ws.transport.http/src/io/openliberty/http/netty/channel/LoggingRecvByteBufAllocator.java [hunk 2]
```diff
@@ -34,38 +34,62 @@ public class LoggingRecvByteBufAllocator implements RecvByteBufAllocator{
...
+    @SuppressWarnings("deprecation")
     public Handle newHandle() {
         return new LoggingHandle(delegate.newHandle());
     }
 
+    /**
+     * A handle that {@link #allocate(ByteBufAllocator)} call in order to log read operations.
+     * Delegates all other operations to the wrapped {@link ExtendedHandle}.
+     * 
+     * Netty has not provided a non-depracated alternative to some calls. In the
```
Comment (pnicolucci) [inline dev/com.ibm.ws.transport.http/src/io/openliberty/http/netty/channel/LoggingRecvByteBufAllocator.java]: non-deprecated

File: dev/com.ibm.ws.transport.http/src/io/openliberty/http/netty/channel/LoggingRecvByteBufAllocator.java [hunk 3]
```diff
@@ -34,38 +34,62 @@ public class LoggingRecvByteBufAllocator implements RecvByteBufAllocator{
...
+         * @param context the channel handler context
+         */
         public void setChannelHandlerContext(ChannelHandlerContext context){
             this.context = context;
         }
 
+        /**
+         * Logs the read request, then delegates allocation.
+         *
+         * @param allocator the {@link ByteBufAllocator}
```
Comment (pnicolucci) [inline dev/com.ibm.ws.transport.http/src/io/openliberty/http/netty/channel/LoggingRecvByteBufAllocator.java:86]: Is `allocator` what you want to use here?

Comment (pnicolucci) [reply dev/com.ibm.ws.transport.http/src/io/openliberty/http/netty/channel/LoggingRecvByteBufAllocator.java:86]: The param in the javadoc doesn't look to match what the method signature contains `public void reset(ChannelConfig config){`

Comment (mrsaldana) [reply dev/com.ibm.ws.transport.http/src/io/openliberty/http/netty/channel/LoggingRecvByteBufAllocator.java:86]: This is actually for the allocate(ByteBufAllocator) on line 90, the git diff looks a bit weird, but it should be correct.

**Review by pnicolucci — APPROVED** [review]

**Pull request merged**